docs/getting-started/scopes.md

```diff
@@ -13,4 +13,10 @@ The following table describes the scopes and a brief description available in th
 |:----------------------------------------------|:-------------------------------------|
 | com.intuit.identity.user.roles.read           | Grants access for reading user role  |
 | qb.company.read                               | Grants access to company information |
-| ...                               | ... |
\ No newline at end of file
+| qb.company.taxidentifier.read                 | Grants access to company tax information (sensitive) |
```

Comment:
tax identifier is the SSN, but it is generic so we can use the Canadian social (SIN) as well. So I would say tax identifier (such as SSN or SIN)

Reply:
that's for employee, company is EIN (US) or CBN Canadian Business Number

Reply:
Awesome thanks for the clarification on this, updated the description